docs/src/components/mui/Chip.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import { Chip as MuiChip } from '@mui/material';
+import { action } from '@storybook/addon-actions';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Icon } from '@ui-kit-2022/components';
+import * as React from 'react';
+
+const ICON_NAMES = Object.keys(Icon);
+const ICON_MAPPINGS = Object.fromEntries(
+  Object.entries(Icon).map(([k, v]) => [k, React.createElement(v)]),
+);
+
+export default {
+  title: 'Material UI/Chip',
```

Comment:
Might want to add `(Pill)` or to the title so show somewhere that chip === pill

Reply:
I think I'm just going to make a Pill component in our components package and within it make use of the Chip component and it's style overrides. There is a special case for stock tickers that it could implement, plus the left/right icon semantics can be improved: for instance Chip calls the right icon the deleteIcon, but that icon might not always be used for deletion in our design system.

Reply:
Makes sense.

So you'll leave this here as "Chip" from MUI and then you'll have a new PR building on this for our Pill?

Reply:
Yeah. I can of course also make this `Material UI/Chip (Pill)` just so it's clear what this can be used for in the design system.